src/OutputChannel.js

```diff
@@ -720,20 +720,24 @@ export class OutputChannel extends EventEmitter {
     // Send note on and, optionally, note off message (if duration is a positive number)
     this.sendNoteOn(note, options);
 
-    if (parseInt(note.duration) > 0) {
-      const noteOffOptions = {
-        time: (Utilities.toTimestamp(options.time) || WebMidi.time) + parseInt(note.duration),
-        release: note.release,
-        rawRelease: note.rawRelease
-      };
-      this.sendNoteOff(note, noteOffOptions);
-    } else if (parseInt(options.duration) > 0) {
-      const noteOffOptions = {
-        time: (Utilities.toTimestamp(options.time) || WebMidi.time) + parseInt(options.duration),
-        release: options.release,
-        rawRelease: options.rawRelease
-      };
-      this.sendNoteOff(note, noteOffOptions);
+    const notes = Array.isArray(note) ? note : [note];
+
+    for(let note of notes) {
+      if (parseInt(note.duration) > 0) {
+        const noteOffOptions = {
```

Comment:
I want to refactor this ... is it worth creating a new function in Utilities for creating NoteOffOptions?

Reply:
I'm not sure this is worth adding to the public API. Perhaps a private method is enough?

Reply:
@djipco private method doesn't really do anything here - you have to pass note options in as an {} anyway and the code just ends up shuffled around rather than clearer.